### ISSUE TITLE
Gestures stops working in HorizontalPager with Compose 1.5.0

### ISSUE THREAD

**Issue by usuiat** [usuiat/Zoomable#74]
Modifier.zoomable stops working after swipeing the page of HorizontalPager.

With compose 1.5.0, this issue occurs. (I confirmed with 1.5.0-beta02)
With compose 1.4.3, it does not occur.

This may be a problem on the compose library side, but I will also investigate if it is possible to work around it on the Zoomable side.
If you know more information please let me know.

## Related issues
- #64 
- #73

**Comment by usuiat** [comment]
I found this issue is opened in Google Issue Tracker
https://issuetracker.google.com/issues/282129984

**Comment by usuiat** [comment]
When zoom does not work, the CoroutineScope is no longer active. Therefore `zoomState.applyGesture()` is not called.

https://github.com/usuiat/Zoomable/blob/1802acd982535a005f4d30996b6acb53cd02cf25/zoomable/src/main/java/net/engawapg/lib/zoomable/Zoomable.kt#L234-L253

**Comment by devcarmelo** [comment]
The issue is caused by changes to Modifier.pointerInput() that were made to improve performance, the following article explains the changes: https://medium.com/androiddevelopers/improvements-and-changes-to-composes-pointer-input-6026904ac972

**Comment by usuiat** [comment]
I have released v1.5.0-beta1.
https://github.com/usuiat/Zoomable/releases/tag/v1.5.0-beta1
I have solved this issue, so please try it.

**Comment by usuiat** [comment]
I have released v1.5.0.
https://github.com/usuiat/Zoomable/releases/tag/v1.5.0
So I close this issue.

**Comment by xiaowei-lei** [comment]
If your project cannot using the latest version, try wrap with `key() {}`:
```
HorizontalPager(
    state = state,
    modifier = Modifier.fillMaxSize(),
) {
    key(state.settledPage) {   // Here
        val zoomState = rememberZoomState()
        YourImage(
              modifier =  Modifier.fillMaxSize().zoomable(zoomState = zoomState, onTap ={ ... }),
         ),
        )
    }
}
```
